src/interpreter/transaction.rs

```diff
@@ -52,7 +52,7 @@ impl<S> Interpreter<S> {
             .tx
             .witnesses()
             .get(b as usize)
-            .ok_or(PanicReason::OutputNotFound)
+            .ok_or(PanicReason::WitnessNotFound)
```

Comment:
This looks like an orthogonal bugfix. Ideally make a separate PR for this.

Reply:
Thats true, I just spotted this nit while debugging